targets/esp8266/user/jerry_extapi.c

```diff
@@ -73,9 +73,20 @@ DELCARE_HANDLER(print) {
         printf("%s ", buffer);
         free (buffer);
       }
-      else
+      else if (jerry_value_is_number (args_p[cc]))
       {
-        printf ("(%d) ", args_p[cc]);
+        double number = jerry_get_number_value (args_p[cc]);
+        if ((int) number == number)
+        {
+          printf ("%d", number);
```

Comment:
Does not this cause a warning here? Is it prints a correct number at all? Not just rubbish?

Reply:
I tested it, and it caused an issue. Thanks for your suggestion, I'm going to fix it.

Reply:
Fixed, now it works as intended.